package.json

```diff
@@ -9,22 +21,17 @@
     "cypress-intellij-reporter": "^0.0.6",
     "cypress-plugin-tab": "^1.0.5",
     "eslint": "^8.23.0",
+    "eslint-config-prettier": "^8.5.0",
```

Comment:
You are including prettier, but then not using the plugin at all. We either need to use the prettier rules, or don't require the package. I'd suggest we take it out for now, as all it does is prettifying the code, not finding errors in it. Also, the output from prettier is a bit useless, as every error is categorised as `prettier/prettier`. It also wants us to put semi-colons on the end of every line, which I'd argue is also pointless. So I'd take it out altogether.

Reply:
I stopped using prettier for now because it reported an enormous amount (over 1000) of issues which we'll have to turn off in the config file.  I'll take another look at whether its possible with just a few lines, to turn those off in the config file. I'm not exactly confident short-term that we can get this legacy code to satisfy prettier , due to conflicting requirements, but I guess time will tell.

Reply:
This is now done